lib/core/importer.ts

```diff
@@ -90,10 +91,13 @@ export class Importer {
    */
   private catchOverlappedScanScope(value: { new (...args: any[]): any; name: string }, pathName: string) {
     locate(value as any).then(({ path }: { path: string }) => {
+      this.classMap.set(path, value);
       if (!this.rootPath) this.rootPath = pathName;
       if (this.rootPath !== path) {
         new Logger('ExceptionHandler').error(
-          `ComponentScan() module scope cannot be overlapped.\n\nPotential causes:\n- An overlapped dependency between modules.\n- Please check the module in '${this.rootPath}' and '${path}'\n\nScope [${value.name}]`,
+          `ComponentScan() module scope cannot be overlapped.\n\nPotential causes:\n- An overlapped dependency between modules.\n- Please check the module in '${
+            this.rootPath
+          }' and '${path}'\n\nScope [${value.name} -> ${this.classMap.get(this.rootPath)!.name}]`,
```

Comment:
I know some people may dislike `!` syntax. But in this case, `this.rootPath` key is definetely defined, so I used it to make the code short and clear.
Looks fine to me, but if you have a different opinion please let me know. 😄

Reply:
LGTM